ServerCore/Pages/Teams/Create.cshtml.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using ServerCore.DataModel;
+using ServerCore.ModelBases;
+
+namespace ServerCore.Pages.Teams
+{
+    public class CreateModel : EventSpecificPageModel
+    {
+        private readonly ServerCore.Models.PuzzleServerContext _context;
+
+        public CreateModel(ServerCore.Models.PuzzleServerContext context)
+        {
+            _context = context;
+        }
+
+        public IActionResult OnGet()
+        {
+            return Page();
+        }
+
+        [BindProperty]
+        public Team Team { get; set; }
+
+        public async Task<IActionResult> OnPostAsync()
+        {
+            if (!ModelState.IsValid)
+            {
+                return Page();
+            }
+
+            Team.Event = Event;
```

Comment:
It might be good to prevent multiple teams from having the same name in the same event. This could be a good chance to set the pattern for input validation.

Reply:
This is beyond my current skill set. I mean I know how to tell that it's a duplicate but I don't know how to communicate that on the web page through validation.

Reply:
https://docs.microsoft.com/en-us/ef/core/modeling/relational/unique-constraints

Reply:
Ok, don't treat this as blocking. @asyasky, maybe you can think about how we should deal with this? A unique constraint would keep the DB from accepting it, but wouldn't' be user-friendly (it would probably just throw an exception when you try to put the data in)

Reply:
The code sample in that link does not appear to follow the conventions described immediately above it - either that or I just don't understand the terms well enough. I'm going to pass on this for now, we can discuss in person.